server/aws/modules/metrics/ec2_data_module.tf

```diff
@@ -0,0 +1,8 @@
+module "metrics_data_vm" {
+  source          = "github.com/cds-snc/terraform-ec2-module"
```

Comment:
I tagged the module in github so we can work on it without drifting our other environments. 

can you add `?ref=v1.0.0` to the end of this url

Reply:
Pushed